packages/hapi-asap/src/middleware.ts

```diff
@@ -4,6 +4,9 @@ import {
   createAsapAuthenticator,
   AuthenticatorOptions,
 } from '@ordermentum/asap-core';
+import { getAdminAuth } from '@ordermentum/auth-driver';
+import { HapiAuthPluginOptions, createHapiAuthPlugin } from '@ordermentum/auth-middleware';
```

Comment:
@ish-bindra think this seems worthwhile directionally, but does 1. make a hard dependency on a closed source package on an open source package, 2. is very ordermentum specific

how do you suggest we could approach both those?

Reply:
Hmm, tbh i didn't realise this was open source. 
Think the better approach would be to go the other way around and utilise asap strategy through the auth middleware.